src/js/media-poster-image.js

```diff
@@ -13,20 +13,18 @@ template.innerHTML = `
       pointer-events: none;
       display: inline-block;
       box-sizing: border-box;
-
-      width: auto;
-      height: auto;
     }
 
     img {
       max-width: 100%;
       max-height: 100%;
       min-width: 100%;
       min-height: 100%;
-      background-size: contain;
-      background-position: center;
       background-repeat: no-repeat;
-      object-fit: contain;
+      background-position: var(--media-background-position, var(--media-object-position, center));
+      background-size: var(--media-background-size, var(--media-object-fit, contain));
+      object-fit: var(--media-object-fit, contain);
+      object-position: var(--media-object-position, center);
```

Comment:
default object position is `50% 50%`, not sure if there's a difference between that and center, though.

Reply:
yeah, can't tell a difference between them, so, probably equivalent.

Reply:
yup it's same, checked it too